src/pages/popup/index.css

```diff
@@ -1,6 +1,32 @@
 @tailwind base;
 @tailwind components;
 @tailwind utilities;
+@font-face {
```

Comment:
Do we want these in `@layer base`? 
https://tailwindcss.com/docs/font-family#customizing-your-theme